index.html

```diff
@@ -66,6 +66,7 @@ <h2>Links:</h2>
 			<li>Server's infra source code: <a target="_blank" rel="noopener" href="https://github.com/chaotic-aur/toolbox">GitHub</a></li>
 			<li>Status page: <a target="_blank" rel="noopener" href="https://status.chaotic.cx">Status monitoring</a></li>
 			<li>Latest logs: <a target="_blank" rel="noopener" href="https://builds.garudalinux.org/repos/chaotic-aur/logs/">Logfiles</a></li>
+			<li>Chaotic keyring: <a target="_blank" rel="noopener" href="https://aur.chaotic.cx/chaotic.gpg">Logfiles</a></li>
```

Comment:
Last change: "Chaotic keys"
(we don't want people to confounding the keyring package with this file)